src/main/java/org/checkerframework/specimin/InheritancePreserveVisitor.java

```diff
@@ -25,20 +28,20 @@ public InheritancePreserveVisitor(Set<String> usedClass) {
   }
 
   /**
-   * Return the current set of used classes.
+   * Return the set of classes to be added to the list of used classes.
    *
-   * @return The set of used classes.
+   * @return The value of addedClass.
    */
-  public Set<String> getUsedClass() {
-    return usedClass;
+  public Set<String> getAddedClass() {
```

Comment:
The name of this method is a bit misleading, since it returns a set but the method name is singular. (Admittedly, this issue is inherited from the previous version of this method, but we should fix it while we're here.) I'd prefer `getAddedClasses()`, and I also suggest that we rename the field accordingly.